### PR TITLE
app initial setup

### PR DESCRIPTION
**Pull request by bulic**

### CODE REVIEW

File: Exebite.API/appsettings.json [hunk 1]
```diff
@@ -14,6 +14,6 @@
 
   },
   "ConnectionStrings": {
-    "ExeBiteConnectionString": "Server=[SERVER-NAME];Database=Exebite;Trusted_Connection=True;"
+    "ExeBiteConnectionString": "Server=NB189;Database=Exebite;Trusted_Connection=True;"
```
Comment (itesanovic) [inline Exebite.API/appsettings.json:17]: It is not recommended for local configuration to be committed to git therefore this should be reverted.

File: Exebite.API/credentials.json [hunk 2]
```diff
@@ -0,0 +1,12 @@
+{
+  "type": "service_account",
```
Comment (itesanovic) [inline Exebite.API/credentials.json:2]: It is not recommended for local configuration to be committed to git therefore this file should be removed.

File: Exebite.GoogleSheetAPI/Properties/Resources.resx [hunk 3]
```diff
@@ -118,16 +118,16 @@
...
-    <value>1p95HwHCwbx80L3aWXsStkGbmNfYgG5JOvRvONm0JUXI</value>
+    <value>1J9Q5MbLg66aT22QCjjX7InlKK8b-OXeZHWXjmUenrrc</value>
   </data>
   <data name="LIPA" xml:space="preserve">
-    <value>1FJt865bBn69jsbRpyVTqQAFarUjvsiKsom1c2JfR0Hk</value>
+    <value>1GjiSV7n-wivAUaiGQl0SOjDkKMY29SvgFTLfzfPVHiQ</value>
   </data>
   <data name="MIMAS" xml:space="preserve">
-    <value>1YQ6BH2vOmGkxNgDWzct_j3Dznql0OWrwi1nITrtlZHA</value>
+    <value>1kaB79osJftUkw5s8RchsJrB2PUqcrOwyxS2PXFIlap0</value>
```
Comment (itesanovic) [inline Exebite.GoogleSheetAPI/Properties/Resources.resx:130]: It is not recommended for local configuration to be committed to git therefore this should be reverted.

File: Exebite.API/Logs/internal-nlog.txt [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+2020-05-25 09:17:07.8210 Info Closing old configuration.
+2020-05-25 09:17:07.8364 Info Logger has been closed down.
+2020-05-25 09:17:07.8364 Info Shutting down logging...
+2020-05-25 09:17:07.8364 Info Logger has been shut down.
+2020-05-26 09:42:01.6687 Info Message Template Auto Format enabled
+2020-05-26 09:42:01.6820 Info Loading assembly: NLog.Web.AspNetCore
+2020-05-26 09:42:01.7626 Info Adding target File Target[allfile]
+2020-05-26 09:42:01.7816 Info Adding target File Target[ownFile-web]
+2020-05-26 09:42:01.7926 Info Configured from an XML element in C:\Users\Bulic\exebite\Exebite.API\bin\Debug\netcoreapp3.1\nlog.config...
+2020-05-26 09:42:01.8077 Info Found 45 configuration items
```
Comment (stanislavgrujic) [inline Exebite.API/Logs/internal-nlog.txt]: Please do not commit log files to Git.

**Pull request merged**